amlb/runners/container.py

```diff
@@ -93,7 +93,7 @@ def run(self, tasks: Union[str, List[str]] = None, folds: Union[int, List[int]]
             job = self._make_container_job(tasks, folds)
             try:
                 results = self._run_jobs([job])
-                return self._process_results(results, task_name=tasks)
+                return self._results_summary(self._process_results(results))
```

Comment:
Since `self._process_results(results)` can do some work, I'd prefer it on a separate line.
```suggestion
                scoreboard = self._process_results(results)
                return self._results_summary(scoreboard)
```